src/player.cpp

```diff
@@ -5285,8 +5285,8 @@ void player::use( item_location loc )
 
     } else if( used.is_book() ) {
         // TODO: Remove this nasty cast once this and related functions are migrated to avatar
```

Comment:
There is no type cast now.

Reply:
i can reword the comment, though i do want that to go away too eventually